doc/api/deprecations.md

```diff
@@ -94,14 +94,6 @@ Within the [`child_process`][] module's `spawn()`, `fork()`, and `exec()`
 methods, the `options.customFds` option is deprecated. The `options.stdio`
 option should be used instead.
 
-<a id="DEP0007"></a>
-### DEP0007: cluster worker.suicide
```

Comment:
-1 on this bit. Per the deprecation policy, the deprecation code is static and permanent and should remain in the documentation. The `Type:` line should be changed to `End-of`-Life`.